Data/Alias/alias.txt

```diff
@@ -142,7 +142,7 @@
 .left Left closed traffic, maintain VFR not above $1 ft, squawk $squawk, report downwind
 .right Right closed traffic, maintain VFR not above $1 ft, squawk $squawk, report downwind
 ;CENTER
-.unicom No further ATC available, monitor unicom 122.80, good day.
+.advisory No further ATC available, monitor advisory, good day.
```

Comment:
Should we still include 122.8 or is that implied already

Reply:
@UraniumLover

Reply:
included